docs/orchestration/getting-started/basic-core-flow.md

```diff
@@ -0,0 +1,33 @@
+# Basic Core Flow
+
+To get started, paste the code above into an interactive Python REPL session: 
+
+```python
+import prefect
+from prefect import task, Flow
+
+@task
+def hello_task():
+logger = prefect.context.get("logger")
```

Comment:
need to indent these two lines

Reply:
This code block still needs some formatting, tabs and some newline spacing